### DISCUSSION TITLE
motorized faders

### DISCUSSION THREAD

**Discussion by RedApCi** [shanteacontrols/OpenDeck#37] [Ideas]
Hi, opendeck looks awesome! Just what I was looking for to make a controller for FL studio. Are motorized faders supported?
Another feature that would be cool is if it could output keyboard macros to control DAW features.

**Comment by paradajz** [comment]
Hi, thank you. MF aren't supported yet. It's on my TODO list but really it's low priority. Maybe I'll implement something this year.
Can you elaborate a bit more on keyboard macros?

**Reply by RedApCi** [reply]
RE: keyboard macros, it would be cool if there were a way to link a button on a control surface to a typing keyboard output. I.e. I press a physical button on the controller and it sends the keyboard output "ctrl + z" to undo inside my DAW. I guess the better solution for me would be to send midi from OpenDeck and then find some way to convert midi to keyboard output inside the computer.

**Reply by RedApCi** [reply]
I have found a solution using autohotkey https://github.com/genmce/AHK_Generic-Midi-Program
